zanata-war/src/main/webapp/WEB-INF/layout/admin/contact_admin_modal_anonymous.xhtml

```diff
@@ -58,10 +58,10 @@
       <div class="modal__content l--pad-all-1">
         <span>#{msgs['jsf.AdditionalInfo']}</span>
         <span class="txt--meta bx--block">#{msgs['jsf.passwordReset.google.AdditionalInfoMessage']}</span>
-        <s:decorate id="messageField" template="/WEB-INF/layout/field.xhtml">
+        <zanata:decorate id="messageField">
```

Comment:
Only if you want to a4j on blur event. It needs to know which id to
re-render. This one doesn't have a4j in there so not needed.

On Wed, Sep 23, 2015 at 2:08 PM, Alex Eng <notifications@github.com> wrote:

> In
> zanata-war/src/main/webapp/WEB-INF/layout/admin/contact_admin_modal_anonymous.xhtml
> <https://github.com/zanata/zanata-server/pull/928#discussion_r40169701>:
>
> > @@ -58,10 +58,10 @@
> >        <div class="modal__content l--pad-all-1">
> >          <span>#{msgs['jsf.AdditionalInfo']}</span>
> >          <span class="txt--meta bx--block">#{msgs['jsf.passwordReset.google.AdditionalInfoMessage']}</span>
> > -        <s:decorate id="messageField" template="/WEB-INF/layout/field.xhtml">
> > +        <zanata:decorate id="messageField">
>
> is componentId required here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/928/files#r40169701>.
>



-- 
Patrick Huang
Senior Software Engineer
Engineering - Internationalisation
Red Hat, Asia-Pacific Pty Ltd
Level 1, 193 North Quay
Brisbane 4000
Office: +61 7 3514 8278
Fax: +61 7 3514 8199
IRC: pahuang
github: github.com/huangp
Website: www.redhat.com